sql/core/src/test/scala/org/apache/spark/sql/errors/QueryCompilationErrorsSuite.scala

```diff
@@ -168,4 +172,115 @@ class QueryCompilationErrorsSuite extends QueryTest with SharedSparkSession {
       "The feature is not supported: " +
       "Pandas UDF aggregate expressions don't support pivot.")
   }
+
+  test("NO_HANDLER_FOR_UDAF: No handler for UDAF error") {
+    val functionName = "myCast"
+    withUserDefinedFunction(functionName -> true) {
+      sql(
+        s"""
+          |CREATE TEMPORARY FUNCTION $functionName
+          |AS 'org.apache.spark.sql.errors.MyCastToString'
+          |""".stripMargin)
+
+      val e = intercept[AnalysisException] (
+        sql("SELECT myCast(123) as value")
+      )
+
+      assert(e.errorClass === Some("NO_HANDLER_FOR_UDAF"))
+      assert(e.message ===
+        s"No handler for UDAF 'org.apache.spark.sql.errors.MyCastToString'. " +
+          s"Use sparkSession.udf.register(...) instead.")
```

Comment:
s is not needed. Also adjust indentation of the second line.

Reply:
Thanks, will fix.